src/client/components/header/index.scss

```diff
@@ -0,0 +1,166 @@
+$boxshadow: rgba(0, 0, 0, .2);
+$navboxshadow: rgba(214, 214, 214, .5);
+$teamname: #d93738;
+$searchname: #868686;
+$navbackgroundcolor: #fff;
+$uploadnamecolor: #868686;
+
+.rootbody {
+  width: 100%;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  align-items: center;
+  background-color: $navbackgroundcolor;
+  box-shadow: 0 2px 7px 0 $navboxshadow;
+  display: flex;
+  height: 4.375em;
+  width: 100%;
+
+  form {
+    align-items: center;
+    display: flex;
+    width: 100%;
+  }
+
+  .search-name {
+    line-height: 47px;
+    background-color: rgb(255, 255, 255);
+    width: 100%;
+  }
+
+  .team-name {
+    color: $teamname;
+    font-family: 'Roboto', sans-serif;
```

Comment:
do not define font family for each element, because it will be painful if we change our design